source/features/useful-forks.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'dom-chef';
+import elementReady from 'element-ready';
+import * as pageDetect from 'github-url-detection';
+import {RepoForkedIcon} from '@primer/octicons-react';
+
+import features from '.';
+import {getRepo} from '../github-helpers';
+import looseParseInt from '../helpers/loose-parse-int';
+
+async function init(): Promise<void | false> {
+	const forkCount = looseParseInt((await elementReady('.social-count[href$="/network/members"]', {waitForChildren: false}))!);
+	if (forkCount === 0) {
+		return false;
+	}
+
+	const downloadUrl = new URL('https://useful-forks.github.io');
+	downloadUrl.searchParams.set('repo', getRepo()!.nameWithOwner);
+
+	const selector = pageDetect.isRepoForksList() ? '#network' : '#repo-content-pjax-container h2';
```

Comment:
If you just use the merged selector, does it work? 
```suggestion
	const selector = '#network, #repo-content-pjax-container h2';
```

Reply:
@payne911 please check.

I would want 
```suggestion
	const selector = [
		'#network', //isRepoForksList
		'#repo-content-pjax-container h2' //isRepoNetworkGraph
	].join()
```

Reply:
I'm not sure I understand those suggestions. `#network` appears in both pages: this seems unsafe to me.

I just tested it, and somehow `isRepoNetworkGraph` still ends up ignoring the `#network` and going for the `#repo-content-pjax-container h2` appropriately, but I've no clue why that is happening. If it used `#network`, the button would not be placed at the same position as it is now.

Reply:
(As an unrelated side note, I don't like that linting rule which doesn't allow me to adopt the cleaner ternary operator syntax that I prefer because it seems more readable)

```typescript
cond()
  ? whenTrue()
  : whenFalse();
```

Reply:
> I'm not sure I understand those suggestions. `#network` appears in both pages: this seems unsafe to me.

So then this cannot be done. Its good the way it is

Reply:
I had tried to explore a CSS selector which would be valid in both cases. I found one, but it was really ugly, hacky, and susceptible to breaking rapidly in terms of UI changes GitHub might make.

One possible other CSS selector idea I had in mind, I just didn't know if it was even possible. It was something along the lines of
> "from `#network`, take the parent `div`, and then select the first child `div` of that parent"

But that's also ugly, hacky, and britle.

Reply:
> somehow `isRepoNetworkGraph` still ends up ignoring the `#network` and going for the `#repo-content-pjax-container h2` appropriately

To explain what's happening: `querySelector` (the API under `select`) just returns the first element that matches the selector. In this case, even if both are found on the page, probably `#repo-content-pjax-container h2` appears _earlier_ in the dom, therefore "#network" appears to be _ignored_